src/allocator.h

```diff
@@ -113,10 +114,12 @@ typedef struct allocator_ops {
 
    get_size_fn get_capacity;
 
+   get_num_fn get_extent_num;
+
    assert_fn assert_noleaks;
 
    print_fn print_stats;
-   print_fn debug_print;
+   print_fn print_allocated;
```

Comment:
This is a pending cleanup that I was supposed to under SHA d585788, but didn't get around to it then. Am taking this opportunity to name the virtual functions consistently with the called function.